client/app/containers/tests-runs/tests-runs-search/tests-runs-search.controller.js

```diff
@@ -175,7 +218,10 @@ const TestsRunsSearchController = function TestsRunsSearchController(windowWidth
                 testsRunsService.deleteSearchParam(name);
             }
         });
-        onApply();
+        vm.onApply();
+        if (vm.searchParams.query === '' && !testsRunsService.isModalSearchActive() && vm.isMobile()) {
```

Comment:
Why we need do reset in this case?

Reply:
it was necessary, but I see now that after rebase smth was changed on master branch so this case was fixed and now it is no need in this